bindings/py/nupic/bindings/__init__.py

```diff
@@ -21,5 +21,4 @@
 
 """
 ## @file
-"""
-__import__('pkg_resources').declare_namespace(__name__)
+"""
```

Comment:
Hi @abhinavrai44 thanks for helping. 

See the <img width="25" alt="screen shot 2017-01-09 at 2 07 02 pm" src="https://cloud.githubusercontent.com/assets/15566/21785570/eef8dcf8-d674-11e6-8654-ad1e60380979.png"> icon above? That is telling you there is no newline character at the end of this file. It is a good practice to end each file with a newline character. You can do so by placing your cursor at the end of the file and simply pressing "enter" or "return" on your keyboard.